client/landing/gutenboarding/hooks/use-last-location.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useHistory } from 'react-router-dom';
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY as ONBOARD_STORE } from '../stores/onboard';
+
+export default function useLastLocation(): { goLastLocation: () => void } {
+	const history = useHistory();
+
+	const lastLocation = useSelect( ( select ) => select( ONBOARD_STORE ).getLastLocation() );
```

Comment:
Should we add the second argument for `useSelect` ?

```suggestion
	const lastLocation = useSelect( ( select ) => select( ONBOARD_STORE ).getLastLocation(), [] );
```